mappings/net/minecraft/world/gen/feature/StructureSpacingConfig.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_5311 net/minecraft/world/gen/feature/StructureSpacingConfig
```

Comment:
I personally favor name from https://github.com/FabricMC/yarn/pull/1384/files#diff-e032b9ef76cdc917941321473aec9056R1 better.

Reply:
What do you think of the new names I put in the PR?